config/waagent.conf

```diff
@@ -90,3 +90,8 @@ OS.OpensslPath=None
 
 # Determine the update family, this should not be changed
 # AutoUpdate.GAFamily=Prod
+
+# Determine if the overprovisioning feature is enabled. If yes, block extension
+# handling until inVMArtifactsProfile.OnHold is false.
+# Default is disabled
+OverProvisioning.EnableExtensionHandlingBlock=n
```

Comment:
Maybe it's just me, but this seems confusing. "Enabling a block" seems unnecessary when we could simply "disable overprovisioning".